src/node/node.h

```diff
@@ -20,6 +23,9 @@ class ChainClient;
 //! and linking outside of bitcoind for simulation or testing.
 struct Node
 {
+    std::unique_ptr<CConnman> connman;
```

Comment:
I think we should be using `struct` only for the dumbest data structures and not complex classes (such as the node). So this should be changed to `m_connman`

Reply:
Thanks for clarifying. Could add that sentence to the doxygen comment of `Node` for people like me?